### ISSUE TITLE
Typing

### ISSUE THREAD

**Issue by ZanSara** [addisonElliott/pyqt5ac#23]
As a suggestion, it would be nice to introduce typing, at least in function signatures. 

Let me know if you need help on this, but it should be a really small thing.

**Comment by zkovari** [comment]
Yes, I agree. I plan to make a big restructuring for 2.0.0, as the current main method is a bit heavy. Introducing typing would add even more benefit at that point.
